### ISSUE TITLE
How to launch a tale shared with me?

### ISSUE THREAD

**Issue by ludaesch** [whole-tale/whole-tale#29]
I got this URL to a tale from a summer intern:
https://dashboard.wholetale.org/tale/view/5b5258ea715d3400011e9f71
(1) Clicking on the link shows a page that doesn't seem to allow to launch the tale. 
(2) Similarly, the "Back" button doesn't work for this page.

**Comment by Xarthisius** [comment]
For MVP version they need to make it public, then it will appear in your view. We disabled sharing with individual people and groups for now.

**Comment by Xarthisius** [comment]
BTW, I don't see a Tale with this `id` on the system at all.

**Comment by ludaesch** [comment]
What happens if you follow the link? Seokki (the author) tells me that the
tale was made public.
Is it maybe another URL that needs to be shared?

**Comment by Xarthisius** [comment]
It goes to a page that doesn't exist. I see two Seokki's Tales in my Browser tab. Have you tried that?

**Comment by ludaesch** [comment]
Yes, those seem to work. (and the question was more about that other URL
brings me to a page that exists but has no launch facility)
